core/src/main/scala/sttp/tapir/Codec.scala

```diff
@@ -231,6 +231,10 @@ object Codec extends CodecExtensions with CodecExtensions2 with FormCodecMacros
     id[Array[Byte], OctetStream](OctetStream(), Schema.schemaForByteArray)
   implicit val inputStream: Codec[InputStream, InputStream, OctetStream] =
     id[InputStream, OctetStream](OctetStream(), Schema.schemaForInputStream)
+  implicit lazy val inputStreamRange: Codec[InputStreamRange, InputStreamRange, OctetStream] =
+    id[InputStreamRange, OctetStream](OctetStream(), Schema.schemaForInputStreamRange)
+  implicit lazy val inputStreamToRange: Codec[InputStreamRange, InputStream, OctetStream] =
```

Comment:
are we using this?

Reply:
(it seems to be a slightly dangerous conversion between eager & lazy reprs)

Reply:
I guess it's not used, but I added it as as an analogy to already existing
```scala
 implicit val file: Codec[FileRange, TapirFile, OctetStream] = fileRange.map(_.file)(f => FileRange(f))
```
Why do we need this one?

Reply:
That's needed to create a file body with type `TapirFile` (alias for `java.io.File`). In the IS case, that's not needed, as we already cover the "raw"input stream case.